webapp/src/js/services/contact-summary.js

```diff
@@ -59,11 +59,11 @@ angular.module('inboxServices').service('ContactSummary',
       return summary;
     };
 
-    return function(contact, reports, lineage) {
+    return function(contact, reports, lineage, targets) {
       return getGeneratorFunction()
         .then(function(fn) {
           try {
-            return fn(contact, reports || [], lineage || []);
+            return fn(contact, reports || [], lineage || [], targets);
```

Comment:
Because it's most likely that the "targets" will be null (most contacts are not chws that generate targets), would it be ok if I default to "null" - so it's also easy to validate in the `contact_summary` function?
`targets` here would be doc (if defined).

Reply:
Right, if it's not an array it makes no sense. Leave it as is.